docs/onboard-public-officers-using-non-se-machines.md

```diff
@@ -1,37 +1,109 @@
+
+  > **Tip**: For more information on the acceptable email formats, refer to the [Audience](#audience) section.
+
+  3. Indicate if you want to onboard your non-GSIB or non-DWP device to SEED and select **I'm not a robot**.
+
+  > **Note**:
+  >- To access SGTS services using a non-GSIB or non-DWP device, you need SEED.
+  >
+  >
+  >- Apart from the GSIB device, you can access SGTS services via a Government Managed Device(GMD). To achieve this, onboard your non-GSIB or non-DWP device to SEED.
+
+  <kbd>![sign-up-submit](assets/images/onboarding/po-non-se/latest-po-sign-up-non-se-gsib-1.png)</kbd>
+
+  4. Click **Submit**. We will send you the TechPass onboarding invitation email.
+
+  > **Additional information**:
+  >- When your request is approved, you'll receive two separate onboarding emails for TechPass and SEED.
+  >- If you have requested for SEED provisioning, you will receive the SEED onboarding email around the next three business days.
+  >- A TechPass account is created for you and will be *pending* to be activated.
+  >- It becomes activated when you [accept the invitation](#step-3-accept-techpass-invitation) and [complete the TechPass onboarding journey](#step-4-onboard-to-techpass).
+  >- The TechPass and SEED onboarding invitation emails are valid for 30 days. Refer to SEED documentation for more information on what to do if your SEED onboarding invitation has expired.
+  >- If you do not onboard to TechPass within 30 days, we will terminate your TechPass account and notify you via email before the termination. You can again sign up via the TechPass portal or request for TechPass and SEED.
+  >- Onboard to TechPass before enrolling your non-GSIB or non-DWP device to SEED.
+  >- If needed, [Set up security verification for WOG account](#step-2-set-up-security-verification-for-the-wog-account).
+
+
+
+  </details>
+
+## Step 2. Set up security verification for the WOG account
 
 <details>
   <summary style="font-size:20px;font-weight:bold">How to set up security verification for WOG account?</summary>
 
   > **Important**:<br>
-  > Public officers need to set up security verification(multi-factor authentication) for their Whole-of-Government(WOG) account if:
-    >- They need to access Singapore Government Technology Stack (SGTS) services and tools from their GMD(internet device onboarded to SEED) device
-    >- Their SG Govt M365 profile is not displayed on their Microsoft Authenticator app.<br><br>
-  > Others, can skip this and proceed to **Step 2. Sign up for TechPass**.
+  > You need to set up security verification (multi-factor authentication) for your Whole-of-Government(WOG) account if:
+    >- You must access Singapore Government Technology Stack (SGTS) services and tools from your GMD device.
+    >- To view your SG Govt M365 profile on the Microsoft Authenticator app.
+  >
+  >  
+  > Others can skip this step and proceed to [**Step 3. Accept TechPass invitation**](#step-3-accept-techpass-invitation).
```

Comment:
just curious why do we make this optional?

Reply:
Is there a chance that users could already have this set up?  if not, i will make it to sound mandatory.

Reply:
so removed from the note as it is already in step2.